### ISSUE TITLE
Bug in dual sorption

### ISSUE THREAD

**Issue by jbrezmorf** [flow123d/flow123d#1409]
1. in commit 9611bb2f8, in SorptionImmob::compute_common_ele_data
  there should be:
  `this->common_ele_data.scale_aqua = por_imm;`
2. all dual porosity tests have porosity_immobile = 0.25, in majority (possibly all) tests this is equal to the total porosity
  that makes mobile porosity inactive. 
3. reduce number of tests dealing with dualporosity:
  - [x] just single test in 03_generic_output to test output of both mobile and immobile concentrations
    remove 05, 07
  - [x] 03-04 - remove, but put default flow output fields to other test, check that we have test of FV sources in 20 or 21 folder
  - [x] 02/12 - remove sorption, test time unit conversion only for flow and solute equations
  - [x] 21/01 - remove, VTK/GMSH output tested in 03_generic_output
  - [x] 22/01 and 27/02 - keep but rearrange to match 21/03 
4. need to update all dependent tests:
  - [x] 26_solute_dg_frac/03_dg_dp_sorp_small.yaml (porosity_imm = porosity_total = 0.25 !!)
  - [x] 26_solute_dg_frac/02_dg_dp.yaml (porosity_imm = porosity_total = 0.25 !!)
  - [x] 21_solute_fv_frac/03_fv_dp_sorp_small.yaml (porosity_imm = porosity_total = 0.25 !!)
  - [x] 21_solute_fv_frac/02_fv_dp_sorp_vtk.yaml (porosity_imm = porosity_total = 0.25 !!)
  - [x] 21_solute_fv_frac/01_fv_dp_sorp_gmsh.yaml (porosity_imm = porosity_total = 0.25 !!)
  - [x] 22_solute_fv_time/01_fv_dp_sorp_small.yaml
  - [x] 27_solute_dg_time/02_dg_dp_sorp_small.yaml (porosity_imm = porosity_total = 0.25 !!) 
  - [x] 03_generic_output/05_fv_dp_sorp_small_output_input_fields.yaml
  - [x] 03_generic_output/06_fv_dp_sorp_small_output_multi_fields.yaml
  - [x] 03_generic_output/07_dg_dp_sorp_small_output_multi_fields.yaml
  - [x] 03_generic_output/04_fv_dp_sorp_small_nobalance.yaml
  - [x] 02_generic_input/12_time_unit_nested_equation.yaml
  - [x] 31_dual_por/*
  - [x] 31_dual_por/*
  
5. check why this made no regression in the tests. The only ref_out changes in that pull request were 22/01 and 22/02

**Comment by Paulie14** [comment]
Regarding point 2: we have only `porosity` and `porosity_immobile`.
The first is used in transport and it it regards to mobile porosity - it is passed to dual porosity model.
The later is set in the dual porosity model.
We have no such thing as `total_porosity` nor we check anything like `porosity+porosity_immobile <= 1`.

**Comment by Paulie14** [comment]
- not sure that you meant for the test 03-04 with the flow output fields
- tests 21-03 and 22-01 difference is only constant vs time dependent porosity

**Comment by Paulie14** [comment]
AD. 5
The bug fix made diff in tests: 22-01, 27-02, where the (mobile) porosity is time dependent, and therefore different from immobile porosity.
It seems that everywhere else (where dual porosity sorption is used) both porosities are the same.

**Comment by Paulie14** [comment]
TODO:
03-04 - alternative of 21-03, remove reaction term, aim: no balance test

**Comment by Paulie14** [comment]
I went to commit 101b53ca43f275304175358be7ab81ec0d58b263, i.e. before DF changes and tested case 22-01.
After fixing the bug with `por_imm` in immobile sorption, the test failed.

The only tests that were affected by this change were: 22-01 and 27-02:
22-01 - ref output was changed back then
27-02 - used to be test 22-02, ref output was changed back then

I do not know, why @dflanderka did not see these tests failing, while fixing the bug.
Both these tests were affected by the bug before DF changes and after..

I also compared the results of the test 22-01 before DF changes and after, with the bug fixed, and the results are equal (up to last 2 digits in concentration..).
I also tried computation without `make_tables` in `sorption_base.cc`, with the bug fixed, and the results were also nearly equal.
The tables were updated on every data change, so it seems that there was no problem.

I do not see any other problems, I suppose the PR #1413 can be merged.
